File: 백준/Bronze/2525. 오븐 시계/오븐 시계.cc
#include <stdio.h>

int main(void) {
	int H, M, time;
	scanf("%d %d %d", &H, &M, &time);
	
	if((M+time)>=60) {
		H += (M+time)/60;
		M = (M+time)%60;
		if(H>=24) {
			H = H%24;
		}
	}
	else {
		M += time;
	}
	printf("%d %d", H, M);
	return 0;
} 

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner num = new Scanner(System.in);

        int H = num.nextInt();
        int M = num.nextInt();
        int time = num.nextInt();

        if((M + time) >= 60) {
            H += (M+time)/60;
            M = (M+time)%60;
            if(H>=24) {
                H = H%24;
            }
        }
        else {
            M += time;
        }

        System.out.println(H + " " + M);
    }
}